types/react-table/index.d.ts

```diff
@@ -362,22 +362,22 @@ export interface ComponentProps {
 
 export interface TextProps {
     /** Default: 'Previous' */
-    previousText: string;
+    previousText: React.ReactNode | React.ComponentType;
 
     /** Default: 'Next' */
-    nextText: string;
+    nextText: React.ReactNode | React.ComponentType;
 
     /** Default: 'Loading...' */
-    loadingText: string;
+    loadingText: React.ReactNode | React.ComponentType;
```

Comment:
Same as for `previousText`. This can only be a `ReactNode`:
https://github.com/react-tools/react-table/blob/master/src/defaultProps.js#L264